instapi/models/user.py

```diff
@@ -88,6 +88,18 @@ def media_count(self) -> int:
         """
         return cast(int, self.user_detail()["media_count"])
 
+    @property
+    def get_phone_number(self) -> str:
```

Comment:
I think a better name will be `phone_number` because it's a property and not a getter method.